### PR TITLE
Update to recent template, add latest terms.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Review by mattcollier — APPROVED** [review]
TY.

File: contexts/age-v1.jsonld [hunk 1]
```diff
@@ -3,9 +3,34 @@
     "@protected": "true",
     "id": "@id",
     "type": "@type",
+    "description": "https://schema.org/description",
+    "image": "https://schema.org/image",
+    "name": "https://schema.org/name",
+    "PersonalPhotoCredential": "https://convenience.org/vocab#PersonalPhotoCredential",
+    "concealedIdToken": {
```
Comment (dlongley) [inline contexts/age-v1.jsonld:10]: Should this be a top-level term? What about overAge and the others?

File: contexts/age-v1.jsonld [hunk 2]
```diff
@@ -3,9 +3,34 @@
...
+          "@id": "https://w3id.org/cit#meta",
+          "@type": "https://w3id.org/security#multibase"
+        },
+        "payload": {
+          "@id": "https://w3id.org/cit#payload",
+          "@type": "https://w3id.org/security#multibase"
+        }
+      }
+    },
+    "anchoredResource": "https://convenience.org/vocab#anchoredResource",
```
Comment (dlongley) [inline contexts/age-v1.jsonld:32]: This should be in the security vocab and use `@type: @id`.

File: contexts/age-v1.jsonld [hunk 3]
```diff
@@ -3,9 +3,34 @@
...
+    "AgeVerificationCredential": "https://convenience.org/vocab#AgeVerificationCredential",
+    "contentHash": "https://w3id.org/security#contentHash"
```
Comment (dlongley) [inline contexts/age-v1.jsonld:34]: We should use multibase here for the type.

File: contexts/age-v1.jsonld [hunk 4]
```diff
@@ -3,9 +3,36 @@
...
     "overAge": {
       "@id": "https://w3id.org/age#overAge",
       "@type": "http://www.w3.org/2001/XMLSchema#positiveInteger"
-    }
+    },
+    "concealedIdToken": {
+      "@id": "https://w3id.org/cit#concealedIdToken",
+      "@type": "https://w3id.org/security#multibase"
+    },
+    "anchoredResource": "https://w3id.org/security#anchoredResource",
```
Comment (dlongley) [inline contexts/age-v1.jsonld:17]:
```suggestion
    "anchoredResource": {
      "@id": "https://w3id.org/security#anchoredResource",
      "@type": "@id"
    },
```

File: contexts/age-v1.jsonld [hunk 5]
```diff
@@ -3,9 +3,36 @@
...
+    "contentHash": "https://w3id.org/security#contentHash",
```
Comment (dlongley) [inline contexts/age-v1.jsonld:18]:
```suggestion
    "contentHash": {
      "@id": "https://w3id.org/security#contentHash",
      "@type": "https://w3id.org/security#multibase"
    },
```

File: contexts/age-v1.jsonld [hunk 6]
```diff
@@ -3,9 +3,42 @@
     "@protected": "true",
     "id": "@id",
     "type": "@type",
+    "description": "https://schema.org/description",
+    "image": "https://schema.org/image",
```
Comment (dlongley) [inline contexts/age-v1.jsonld:7]:
```suggestion
    "image": {
      "@id": "https://schema.org/image",
      "@type": "@id"
    },
```

**Review by dlongley — APPROVED** [review]
See one remaining comment. Thanks!

File: contexts/age-v1.jsonld (hunk 6 above)
Comment (dmitrizagidulin) [reply contexts/age-v1.jsonld]: Changed, thanks.

**Pull request merged**